main/src/main/scala/sbt/Plugins.scala

```diff
@@ -216,12 +216,12 @@ object Plugins extends PluginsFunctions {
             case Right(results) =>
               log.debug(s"  :: deduced result: ${results}")
               val selectedAtoms: List[Atom] = results.ordered
```

Comment:
It does say ordered here, but evidently not ordered enough.

Reply:
I think [not ordered in the Ordering sense](https://github.com/sbt/sbt/blob/54d9e17cad45f985f62871d107a5c78e1b3219db/internal/util-logic/src/main/scala/sbt/internal/util/logic/Logic.scala#L217C5-L217C5).